canonical-web-and-design/maas-ui
# Add placeholder headings to make switching between apps more seamless.

## Done
Provide some markup for headings for both the angular and react apps, so that something appears before the apps are bootstrapped. This should make switching between the apps more seamless, or less jarring at least.

## QA
In development, the react app loads vanilla via a js import, so you won't see this markup nicely styled (however it is styled correctly in production which has a static link to the build css). You should see the angularjs app's header appear with styles however.
